src/MoBi.Presentation/Presenter/EditDistributedParameterPresenter.cs

```diff
@@ -223,7 +223,7 @@ private void addIfNotDefined(string parameterName, IDimension dimension = null)
             return;
 
          var parameter = createDistributionParameter(parameterName, dimension);
-         AddCommand(_parameterTask.AddToParent(parameter, _distributedParameter, BuildingBlock));
+         AddCommand(_parameterTask.AddTo(parameter, _distributedParameter, BuildingBlock));
```

Comment:
I removed the AddToParent and AddToProject to be less weird since we were in many cases not adding to project.